recipe/meta.yaml

```diff
@@ -12,11 +13,21 @@ source:
     # https://gitlab.freedesktop.org/mesa/mesa/-/issues/4729
     - shader_init.patch
 
+{% if build_variant == "qt" %}
+{% set build_num = build_num + 100 %}
+{% endif %}
+
 build:
-  number: 1
+  number: {{ build_num }}
   skip: true  # [win]
+  {% if build_variant == "egl" %}
+  skip: true  # [osx]
+  {% endif %}
+  string: h{{ PKG_HASH }}_{{ build_num }}_{{ build_variant }}
+  track_features:
+    - paraview-egl   # [build_variant == "egl"]
   run_exports:
-    - {{ pin_subpackage('paraview', max_pin='x.x') }}
+    - {{ pin_compatible('paraview', max_pin='x.x') }}
```

Comment:
This should still be `pin_subpackage`

Reply:
I changed it because the CI failed and it suggested to use `pin_compatible`. I will change it back.